knowledge-base/kb-windowsformhost-rendering.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Windows Form Host is Not Rendering If the Panel is Collapsible
+description: "How to render Windows Form host if the Panel is collapsible"
+type: how-to
+page_title: How to render Windows Form Host if the Panel is Collapsible in RadRibbonView for WPF
```

Comment:
Each word in the "title" and "page_title" properties should begin with an uppercase letter.